src/components/Sep31Send.tsx

```diff
@@ -49,76 +84,185 @@ export const Sep31Send = () => {
     setFormData(updatedState);
   };
 
+  const handleTypeChange = (type: CustomerType, typeId: string) => {
+    const updatedTypes = {
+      ...customerTypes,
+      [type]: typeId,
+    };
+
+    setCustomerTypes(updatedTypes);
+  };
+
   const handleSubmit = (
     event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
   ) => {
     event.preventDefault();
     dispatch(submitSep31SendTransactionAction({ ...formData }));
   };
 
+  const handleSelectTypes = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+  ) => {
+    setErrorMessage("");
+    const { sender, receiver } = customerTypes;
+    event.preventDefault();
+
+    if (!sender || !receiver) {
+      setErrorMessage("Sender and receiver types are required");
+      return;
+    }
```

Comment:
A receiving anchor could only require KYC for one of the sending & receiving customers, or neither.

Reply:
I'll add better validation 👍